spec/controllers/events_controller/delete_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::EventsController, type: :controller do
+  let(:current_organization) { create(:organization) }
+  let(:first_organizer) { create(:user, role: :organizer, organization: current_organization) }
+  let(:second_organizer) { create(:user, role: :organizer, organization: current_organization) }
+  let(:organization) { create(:organization) }
+  let(:third_organizer) { create(:user, role: :organizer, organization: organization) }
+  let(:event_of_current_organization) { create(:event, user_id: first_organizer.id) }
+
+  describe 'DELETE #destroy' do
+    context 'when user is organizer of current event\'s organization' do
+      before do
+        @header = second_organizer.create_new_auth_token
+        request.headers.merge!(@header)
+      end
+
+      it 'returns no_content status' do
+        delete :destroy, params: { id: event_of_current_organization.id }
+        deleted_event = Event.find_by(id: event_of_current_organization.id)
```

Comment:
no need in this.
please refer documentation for "expect to change" and "expect not to change"
please update all tests which test creation/destroying

Reply:
@LuginaJulia what about Vlad's comment?

Reply:
forgot to delete it

Reply:
@LuginaJulia to avoid such things as 'I forgot to fix something' you can click button 'Resolve discussion' after each fix